packages/@aws-cdk/aws-lambda-nodejs/parcel-bundler/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM node:13.8.0-alpine3.11
```

```suggestion
ARG VERSION=13.8.0-alpine3.11
FROM node:$VERSION
```

With a prop allowing to customize the version? and the default either in the Dockerfile as above or in the .ts file.

---

Done

---

Can we expose this build arg as an optional prop in `NodejsFunctionProps` and then `BuilderOptions`? There's a risk of being stucked with a parcel-bundler version that could be incompatible with the fixed node/alpine version of the Dockerfile?

---

Done

---

Expose `nodeDockerTag` in `NodejsFunctionProps` and pass it to the `Builder` and we're done here.

---

Done